crates/bevy_render/src/texture/compressed_image_saver.rs

```diff
@@ -30,6 +33,10 @@ impl AssetSaver for CompressedImageSaver {
         compressor_params.set_basis_format(basis_universal::BasisTextureFormat::UASTC4x4);
         compressor_params.set_generate_mipmaps(true);
         let is_srgb = image.texture_descriptor.format.is_srgb();
+        let sampler_descriptor = match &image.sampler_descriptor {
```

Comment:
Not too sure about that one, if we want to use the actual default sampler from the render plugin, it would require either passing its descriptor here or changing `ImageLoaderSettings::sampler_descriptor` to an `ImageSampler` and using `ImageSamplerDescriptor` instead of `wgpu::SamplerDescriptor` in `ImageSampler`.